_pages/en_US/installing-boot9strap-(fredtool-11.15).txt

```diff
@@ -0,0 +1,77 @@
+---
+title: "BannerBomb3-Fredtool (11.15)"
+---
+
+{% include toc title="Table of Contents" %}
+
+{% capture technical_info %}
+<summary><em>Technical Details (optional)</em></summary>
+
+To dump system DSiWare, we exploit a flaw in the DSiWare Data Management window of the Settings application.
+
+To accomplish this, we use your system's encryption key (movable.sed) to build a DSiWare backup that exploits the system to dump the DSi Internet Settings application to the SD root.
+
+Once we have a DSiWare backup, inject it into the DS Internet Settings application.
```

Comment:
This sentence shouldn't be imperative

Reply:
une fois que nous nous avons le backup du dsiware injectez-le à l'appli ds internet settings

Reply:
je ne sais pas ce que tu as dit